tests/test_lite/test_dbapi03.py

```diff
@@ -5,48 +5,52 @@
 
 class TestShutdown:
     def test_commited_on_restart(self, monetdbe_cursor_autocommit):
-        (cursor, connection, dbfarm) = monetdbe_cursor_autocommit
+        context = monetdbe_cursor_autocommit
```

Comment:
these test suites are a bit messy if i remember correctly, sorry about that. I just copied them from the monetdb lite repo and modified them to fit our less-globals-api.